### ISSUE TITLE
Design | CH 31 | Inconsistent Headings

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#68020]
## Issue Description
_CAIA feedback in #64358._

### Staging or Production Link
- [Staging Link](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)

### Prototype Link
- [Mural User Flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1684161597392/5c1f8869cf5d03b792c8b82cf19b39dc26c486c2?sender=u0e8ac1d6d7681bb7e1b80558)

### Test Users
- vets.gov.user+24@gmail.com
- [staging test user repository](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users.csv)

### Details
`<h3>` headings are styled inconsistently across the form. This can be confusing for sighted users, who use heading styles to better perceive hierarchy on the page.

On [step 1](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/veteran-contact-information), the "Mailing Address" heading uses a font utility class to style it as an `<h4>`:
`<h3 class="vads-u-font-size--h4">Mailing Address</h3>`

On step 3, however, the "How would you like us to contact you?" heading doesn't use any font utility classes:
`<h3>How would you like us to contact you?</h3>`

### Reproduction steps
1. [Go to the form](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)
2. Click "Apply for Veteran Readiness and Employment with VA Form 28-19003" and sign in
3. Navigate to Step 1 of the form, and inspect the "Mailing Address" heading
4. Navigate to Step 3 of the form, and inspect the "How would you like us to contact you?" heading

### Proposed solution or next steps
Pick one way to style `<h3>` across the form, and make sure it's consistent. 

Either:
- Use a font utility class, like `<h3 class="vads-u-font-size--h4">Mailing Address</h3>`, **OR**
- Don't use a font utility class, like `<h3>How would you like us to contact you?</h3>`

### References, articles, or WCAG support
[Subheadings and accessibility](https://www.boia.org/blog/subheadings-and-accessibility-what-is-a-visual-heading)

> "To explain why subheadings must be both visual and programmatically determinable, let’s consider a quick example. 
> 
> A website uses subheading tags (such as `<h2>` and `<h3>`) to structure its content. When a screen reader user accesses the website, they can “scan" the text using the subheading tags to find the information they want. 
> 
> However, the text within the subheading tags looks the same as the other text on the page. People who do not use screen readers or other assistive technologies (AT) cannot quickly identify the headings — in order to find information, they’ll need to read through blocks of text. "

---
## Tasks
- [ ] Decide on a heading approach

## Acceptance Criteria
- [ ] Sketch file of updated design
- [ ] Assign to Laura Steele to create follow-on engineering ticket
---
